src/components/tabs/__internal__/tab-title/tab-title.spec.js

```diff
@@ -217,6 +217,48 @@ describe("TabTitle", () => {
     });
   });
 
+  describe("when isInSidebar: true", () => {
```

Comment:
minor point and maybe it's just my opinion but I don't like this test description - `isInSidebar` is from the DraweSidebarContext which is purely an implementation detail of these 2 components.

I'd personally replace this with something like `"when inside a Drawer sidebar"`